db/post_migrate/20210909041448_make_topic_id_nullable_for_bookmarks.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class MakeTopicIdNullableForBookmarks < ActiveRecord::Migration[6.1]
+  def up
+    change_column_null :bookmarks, :topic_id, true
```

Comment:
We need to mark the column as readonly which helps to keep ensure that no one else is writing into this column . We have a helper for this `Migration::ColumnDropper.mark_readonly(table_name, column_name)`

Reply:
Ah yes I forget about these ColumnDropper helpers, thanks.